classes/submission/Repository.inc.php

```diff
@@ -270,6 +295,7 @@ public function validate(?Submission $submission, array $props, array $allowedLo
 
         // The contextId must match an existing context
         $validator->after(function ($validator) use ($props) {
+            // TODO: contextid might be typo. Check.
```

Comment:
Good catch! Fixed in https://github.com/pkp/pkp-lib/commit/cb39a105d82e3696d9c5d0bbe8350a6f5518777a and https://github.com/pkp/pkp-lib/commit/f3df39a77f1df303f8f926bd48c92cb4a1da1b39